daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -11,34 +15,47 @@ module DA.Daml.Helper.Ledger (
     runLedgerAllocateParties,
     runLedgerUploadDar,
     runLedgerFetchDar,
-    runLedgerNavigator
+    runLedgerNavigator,
     ) where
 
-import Control.Exception.Safe (SomeException,Exception,catch,throw)
+import Control.Exception.Safe (Exception, SomeException, catch, throw)
+import Control.Lens (Prism', preview)
 import Control.Monad.Extra hiding (fromMaybeM)
-import DA.Ledger (LedgerService,PartyDetails(..),Party(..),Token)
-import Data.Aeson
-import Data.Aeson.Text
-import Data.List.Extra as List
+import DA.Ledger (LedgerService, Party(..), PartyDetails(..), Token)
 import qualified DA.Ledger as L
+import Data.Aeson ((.=))
+import qualified Data.Aeson as A
+import qualified Data.Aeson.Lens as A
+import Data.Aeson.Text
 import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as BSC
+import Data.List.Extra as List
+import Data.String (IsString)
 import qualified Data.Text as T
 import qualified Data.Text.Lazy as TL
 import qualified Data.Text.Lazy.IO as TL
+import qualified Data.Vector as Vector
+import Network.HTTP.Simple
 import System.Environment
 import System.Exit
 import System.FilePath
 import System.IO.Extra
 import System.Process.Typed
 
 import qualified DA.Daml.LF.Ast as LF
-import DA.Daml.Compiler.Fetch (LedgerArgs(..),runWithLedgerArgs, fetchDar)
+import DA.Daml.Compiler.Fetch (LedgerArgs(..), runWithLedgerArgs, fetchDar)
 
 import DA.Daml.Helper.Util
 import DA.Daml.Project.Util (fromMaybeM)
 
+data LedgerApi
+  = GRPC
```

Comment:
```suggestion
  = Grpc
```
Seems more consistent with HttpJson

Reply:
Changed it. I was coming from 'gRPC', but no big fan of too much upper case either.